README.md

```diff
@@ -74,7 +74,7 @@ To use Device Simulator Express, install the extension from the marketplace and
 
 ### 1. Start with the “New File” Command.
 
-1. Type in Device Simulator Express: New File” in the command palette (`CTRL+SHIFT+P` to open the command palette).  
+1. Type in `"Device Simulator Express: New File”` in the command palette (`CTRL+SHIFT+P` to open the command palette).  

```

Comment:
_&ast;nit&ast;_
Maybe use the same character for opening and closing quotation marks?

Reply:
Completely agree. Updated.